app/controllers/AccountManagement.scala

```diff
@@ -325,12 +327,12 @@ object ManageWeekly extends ContextLogging {
 }
 
 
-class AccountManagement extends Controller with ContextLogging with CatalogProvider with CommonActions {
+class AccountManagement(fBackendFactory: TouchpointBackends) extends Controller with ContextLogging with CommonActions {
```

Comment:
What about 'backendFactory' or 'touchpointBackends' or 'thirdPartyEnvironment'? (I don't like fBackendFactory!)

Reply:
sorry yes I temporarily renamed TouchpointBackends everywhere to that on purpose, so I could use search and replace on "TouchpointBackend" without catching it.  I renamed the class to what I really wanted, but some variable names slipped through.  Updating.